src/main/java/com/example/demo/services/NotificationService.java

```diff
@@ -0,0 +1,35 @@
+package com.example.demo.services;
+
+import com.example.demo.entities.User;
+import jakarta.mail.MessagingException;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+import static com.example.demo.services.EmailService.sendEmail;
+
+@Service
+public class NotificationService {
+    @Value("${auction.owner.subject}")
+    static String ownerSubject;
+    @Value("${auction.owner.text}")
+    static String ownerText;
+
+    @Value("${auction.pretender.subject}")
+    static String pretenderSubject;
+    @Value("${auction.pretender.text}")
+    static String pretenderText;
+
+    @Value("${orer.seller.subject}")
+    static String sellerSubject;
+    @Value("${order.seller.text}")
+    static String sellerText;
```

Comment:
does @Value work with static?

Why not using regular non-static fields and constructor injection? Or constants?